packages/variants/src/variants.ts

```diff
@@ -15,7 +16,28 @@ function makeThemeFromPartials(
 }
 
 /**
- * A variant where the background soft shade of the neutral color. Most other colors remain unchanged.
+ * Sets the variant theme based on which variant is currently applied.
```

Comment:
"Returns the specified variant theme for the given theme"

Reply:
maybe add a note that you definitely should not generate a variant of a variant!